insights/config/specs.py

```diff
@@ -371,6 +371,7 @@
     "sshd_config_perms"         : CommandSpec("/bin/ls -l /etc/ssh/sshd_config"),
     "sssd_config"               : SimpleFileSpec("etc/sssd/sssd.conf"),
     "sssd_logs"                 : PatternSpec(r"var/log/sssd/.*\.log$"),
+    "swift_object_expirer_conf": SimpleFileSpec("etc/swift/object-expirer.conf"),
```

Comment:
You might miss a whitespace before the ":" to keep them aligned.

Reply:
BTW, would you mind renaming the specs of `proxy_server.conf` to `swift_proxy_server_conf` in this PR? since the spec name doesn't affect plugins.